File: CSharp/SpaceVIL/SpaceVIL/Layouts/ItemsLayout.cs
using System.Collections.Generic;
using System.Linq;
using System;
using SpaceVIL.Core;

namespace SpaceVIL
{
    internal sealed class ItemsLayout
    {
        private Guid _id;
        internal Guid Id
		{
			get { return  _id; }
			set { _id = value; }
		}
        internal ItemsLayout(Guid layoutId)
        {
            Id = layoutId;
        }
        List<IBaseItem> items = new List<IBaseItem>();
        internal List<IBaseItem> Items
		{
			get { return items; }
			set { items = value; }
		}
        List<IBaseItem> float_items = new List<IBaseItem>();
        internal List<IBaseItem> FloatItems
        {
            get { return float_items; }
            set { float_items = value; }
        }
    }
}

File: Java/SpaceVIL/spacevil/src/main/java/com/spvessel/spacevil/ItemsLayout.java
package com.spvessel.spacevil;

import java.util.*;

import com.spvessel.spacevil.Core.IBaseItem;

final class ItemsLayout {
    ItemsLayout(UUID layoutId) {
        _id = layoutId;
    }

    private UUID _id;

    UUID getId() {
        return _id;
    }

    void setId(UUID value) {
        _id = value;
    }

    List<IBaseItem> _items = new LinkedList<>();

    List<IBaseItem> getItems() {
        return _items;
    }

    void setItems(List<IBaseItem> value) {
        _items = value;
    }

    List<IBaseItem> _float_items = new LinkedList<>();

    List<IBaseItem> getFloatItems() {
        return _float_items;
    }

    void setFloatItems(List<IBaseItem> value) {
        _float_items = value;
    }

    List<IBaseItem> _dialog_items = new LinkedList<>();

    List<IBaseItem> getDialogItems() {
        return _dialog_items;
    }

    void setDialogItems(List<IBaseItem> value) {
        _dialog_items = value;
    }
}